pyspec/msdial_msms/msdial_msms.py

```diff
@@ -0,0 +1,354 @@
+    split = excelSheets[index].split(os.path.sep)
+    return split[-1]
+
+def openWorkBook(excelSheets, index):
+    """Opens excel workbook of excel sheet chosen from excel sheet list
+            
+    Parameters
+    ----------
+    excelSheets : list
+        A list of all excel files in current directory.
+    index : int
+        index of excel sheets list to extract file name from
+        
+    Returns
+    -------
+    wb
+        Workbook of chosen excel sheet
+    """
+    
+    wb = openpyxl.load_workbook(excelSheets[index])
+    return wb
+
+
+def makeSheet(wb):
+    """Returns first sheet of excel workbook passed in.
+            
+    Parameters
+    ----------
+    wb : workbook 
+        Workbook opened using openpyxl
+    
+    Returns
+    -------
+    Workbook sheet
+        returns first sheet in excel file opened  
+    """
+    
+    sheets = wb.sheetnames
+    sheet = wb[sheets[0]]
+    return sheet
+
+def msmsFound(sheet, currentRow, currentColumn = 8):
+    """Returns True if current row and column 8 of excel sheet value is 'True'
+           
+    Parameters
+    ----------
+    sheet : excel workbook sheet
+        The first sheet in excel workbook opened
+    currentRow : int
+        The current row of the feature that data is being taken from
+    currentColumn : int
+        The column in the MS-Dial export sheet that contains msms true of false.
+        Default value of 8.
+            
+    Returns
+    -------
+    bool
+        True if cell value == 'True', otherwise False
+    """
+    
+    return str(sheet.cell(row = currentRow, column = currentColumn).value) == 'True'
+
+def retentionTimeColumn(sheet):
+    """Returns True if row 4 and column 8 of excel sheet value is 'Average Rt(min)'
+            
+    Parameters
+    ----------
+    sheet : excel workbook sheet
+        The first sheet in excel workbook opened
+            
+    Returns
+    -------
+    bool
+        True if cell value == 'Average Rt(min)', otherwise False
+    """
+    
+    return sheet.cell(row = 4, column = 2).value == 'Average Rt(min)'
+
+def mzColumn(sheet):
+    """Returns True if row 4 and column 3 of excel sheet value is 'Average Mz'
+            
+    Parameters
+    ----------
+    sheet : excel workbook sheet
+        The first sheet in excel workbook opened
+            
+    Returns
+    -------
+    bool
+        True if cell value == 'Average Mz', otherwise False
+    """
+    
+    return sheet.cell(row = 4, column = 3).value == 'Average Mz'
+
+def msmsColumn(sheet):
+    """Returns True if row 4 and column 3 of excel sheet value is 'MS/MS spectrum'
+            
+    Parameters
+    ----------
+    sheet : excel workbook sheet
+        The first sheet in excel workbook opened
+            
+    Returns
+    -------
+    bool
+        True if cell value == 'MS/MS spectrum', otherwise False
+    """
+    
+    return sheet.cell(row = 4, column = 22).value == 'MS/MS spectrum'
+    
+def test_retentionTimeColumn(sheet):
```

Comment:
these tests should be all in the actual test class and not in the code. You always want to have loose coupling.